schedule/adapter.py

```diff
@@ -7,10 +7,11 @@ def course_to_alg_course(course: Course) -> A_Course:
         a_course = A_Course.objects.get(code=course.course_code)
     except A_Course.DoesNotExist:
         a_course = A_Course()
-    a_course.title = course.course_title,
-    a_course.pengRequired = course.pengRequired,
-    a_course.yearRequired = course.yearRequired
-    a_course.save()
+        a_course.code = course.course_code
```

Comment:
I see how this bug crept in - I changed this from get_or_create(code=course.course_code) to .get with an except, but forgot to add in the course code creation.

Reply:
Yeah no worries - a very easy line to miss